BDArmory/Targeting/TargetSignatureData.cs

```diff
@@ -195,6 +195,15 @@ public static TargetSignatureData noTarget
             }
         }
 
+        public static TargetSignatureData DataLinkNoTarget(Vector3 mlPos, float maxRange, float activeRadarRange)
+        {
+            Vector3 targetPos;
+            if (maxRange > (activeRadarRange * 2)) targetPos = mlPos + (maxRange * mlPos.normalized);
```

Comment:
You don't need so many parentheses. Using too many unnecessarily makes code harder to read.